src/lib.rs

```diff
@@ -507,14 +504,25 @@ impl Section {
         }
     }
 
+    /// Constructs a `Section` via a user supplied for simpler one-liners
```

Comment:
you missed a word here, maybe 'callback'?

Reply:
Hah oops.